File: LLM-Translations/GPT-4/codenet/Java/Go/s951567432.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var count int
	fmt.Scan(&count)

	total := 0
	for ix := 1; ix <= count; ix++ {
		total += ix
	}

	fmt.Println(total)
}

// 

File: LLM-Translations/GPT-4/codenet/Java/C/s951567432.c
#include <stdio.h>

int main() {
  int count;
  scanf("%d", &count);
  
  int total = 0;
  for (int ix = 1; ix <= count; ix++) {
      total += ix;
  }
  
  printf("%d\n", total);
  return 0;
}
// 
